### ISSUE TITLE
 About costs

### ISSUE THREAD

**Issue by 3058132083** [vroland/epdiy#260]
![image](https://github.com/vroland/epdiy/assets/52588960/7a258f2b-c95c-41e6-b798-cc3302e15673)
![image](https://github.com/vroland/epdiy/assets/52588960/27b369e2-798a-455b-ad16-389a38849e54)
These two chips only have different output currents in the manual. I would like to know if cheaper TPS65185 can be used instead of TPS651851
![image](https://github.com/vroland/epdiy/assets/52588960/8460d393-82e8-4451-9e45-f18342d6e99a)

**Comment by seishinkouki** [comment]
Roughly, if your panel size lower then 9", tps65185 is acceptable
小尺寸面板可以用65185

**Comment by vroland** [comment]
Indeed, they are functionally equivalent, except that the TPS651851 has better performance. Make sure to get the RSLR version though, because there is also an RGZ which has a larger package and won't fit.

**Comment by martinberlin** [comment]
Interesting issue!  Since epdiy has now also a [hardware repository](https://github.com/vroland/epdiy-hardware/issues) I suggest that further issues about hardware can be discuss directly there.
In my opinion the best would be to make a page in the WiKi section on the hardware side, that can be edited without making commits, etc (Or yes it does but internally and works just like a WiKi Content manager) and there we build a nice table with possible alternatives.
Like @vroland mentioned above the most important thing is that the replacement part has the same footprint.

**Comment by vroland** [comment]
@martinberlin I think putting the wiki page into a markdown file that's also built into the pages website. That makes it easier for people to make a PR for that file to add new suggestions. In my experience, the wiki is also a bit hard to find.
